docs/core/index.md

```diff
@@ -19,9 +19,9 @@ Download the [.NET Core SDK](https://www.microsoft.com/net/download) to try .NET
 
 All .NET Core versions are available at [.NET Core Downloads](https://dotnet.microsoft.com/download/dotnet-core) if you're looking for another .NET Core version.
 
-## .NET Core 3.0
+## .NET Core 3.1
 
-The latest version is .NET Core 3.0. New features include Windows Desktop support with Windows Presentation Foundation (WPF) and Windows Forms, full stack C# web development with Blazor, new enhancements to SignalR and Azure SignalR Service, new C# language features with C# 8, and much more. For a full listing of the new features in .NET Core 3.0, see [What's new in .NET Core 3.0](./whats-new/dotnet-core-3-0.md).
+The latest version is .NET Core 3.1. Which includes minor improvements over .NET Core 3.0. However, .NET Core 3.1 is a long-term supported release. For more information about the .NET Core 3.1 release, see [What's new in .NET Core 3.1](./whats-new/dotnet-core-3-1.md).
```

Comment:
we need to create the what's new still, no?

Reply:
It's here. Git is just having fun with me. Refresh. :)